source/server/lua/loader/LuaDimensionLoader.cpp

```diff
@@ -38,8 +38,34 @@ void LuaDimensionLoader::loadDimension(const sol::table &table) const {
 	if (biomesObject.valid() && biomesObject.get_type() == sol::type::table) {
 		Dimension &dimension = Registry::getInstance().registerDimension(id, name);
 		dimension.setSky(table["sky"].get<std::string>());
-		dimension.setGravity(table["gravity"].get_or(1.f));
 
+		sol::object dimensionPhysicsObject = table["dimension_physics"];
+		if (dimensionPhysicsObject.valid() && dimensionPhysicsObject.get_type() == sol::type::table) {
```

Comment:
Please print an error message with `gkError()` if `dimensionPhysicsObject` is not a table (check other places in the code for examples)

Reply:
Please print the error **only when the object is valid and not a table**. If the object isn't valid, it's fine.